dist/index.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <script src="https://kit.fontawesome.com/86098cddac.js" crossorigin="anonymous"></script>
+    <title>To Do List</title>
+<script defer src="runtime.bundle.js"></script><script defer src="index.bundle.js"></script></head>
+<body>
+  <div class="listContainer">
+    <p>Today's To Do</p>
+    
+    <ul class="lists">
+      <li><input type="text" id="add-input" placeholder="Add new task..." required/><i id="add-btn" class="fa-solid fa-plus"></i></li>
+    </ul>
+    <div class="btn-container">
+    <button type="reset" class="reset-btn">Clear all completed</button>
+    </div>
```

Comment:
[Optional]Why don't you generate this content from javascript? I suggest you do it, it'll be beneficial for your curriculum.